tests/rule_based_profiler/data_assistant/test_onboarding_data_assistant.py

```diff
@@ -145,3 +254,390 @@ def test_onboarding_data_assistant_result_batch_id_to_batch_identifier_display_n
             FULLY_QUALIFIED_PARAMETER_NAME_ATTRIBUTED_VALUE_KEY
         ].keys()
     )
+
+
+def test_onboarding_data_assistant_plot_descriptive_notebook_execution_fails(
+    bobby_columnar_table_multi_batch_deterministic_data_context,
+):
+    context: DataContext = bobby_columnar_table_multi_batch_deterministic_data_context
+
+    new_cell: str = (
+        "data_assistant_result.plot_metrics(this_is_not_a_real_parameter=True)"
+    )
+
+    with pytest.raises(nbconvert.preprocessors.CellExecutionError):
+        run_onboarding_data_assistant_result_jupyter_notebook_with_new_cell(
+            context=context,
+            new_cell=new_cell,
+            implicit=False,
+        )
+
+    with pytest.raises(nbconvert.preprocessors.CellExecutionError):
+        run_onboarding_data_assistant_result_jupyter_notebook_with_new_cell(
+            context=context,
+            new_cell=new_cell,
+            implicit=True,
+        )
+
+
+def test_onboarding_data_assistant_plot_descriptive_notebook_execution(
+    bobby_columnar_table_multi_batch_deterministic_data_context,
+):
+    context: DataContext = bobby_columnar_table_multi_batch_deterministic_data_context
+
+    new_cell: str = "data_assistant_result.plot_metrics()"
+
+    run_onboarding_data_assistant_result_jupyter_notebook_with_new_cell(
+        context=context,
+        new_cell=new_cell,
+        implicit=False,
+    )
+
+    run_onboarding_data_assistant_result_jupyter_notebook_with_new_cell(
+        context=context,
+        new_cell=new_cell,
+        implicit=True,
+    )
+
+
+def test_onboarding_data_assistant_plot_prescriptive_notebook_execution(
+    bobby_columnar_table_multi_batch_deterministic_data_context,
+):
+    context: DataContext = bobby_columnar_table_multi_batch_deterministic_data_context
+
+    new_cell: str = "data_assistant_result.plot_expectations_and_metrics()"
```

Comment:
Any way we can populate notebooks without string values? I had to refactor a decent number of tests as a result of changing our API and it would be nice to leverage IDE renaming if possible.

Reply:
> Any way we can populate notebooks without string values?

Not that I know of. You can always to find/replace cant you?